Dockerfile

```diff
@@ -6,10 +6,13 @@ ARG RAILS_ENV
 ARG DERIVATIVES_PATH
 ARG UPLOADS_PATH
 ARG CACHE_PATH
+ARG WORKING_PATH
```

Comment:
This doesn't seem to be used

Reply:
It is used by Hyrax I think, see:

https://github.com/research-technologies/hyrax_leaf/pull/5/files#diff-39ac5326a57fe21da35b98b056174a3dR14
and
https://github.com/research-technologies/hyrax_leaf/pull/5/files#diff-39ac5326a57fe21da35b98b056174a3dR14

It's seems like there's a lot of these paths, but I thought it useful to separate them out.

Reply:
I create one volume (uploads) and use it for uploads/cache/working_dir

Reply:
Thanks for telling about working_dir. I didn't know that one